cmd/helm/fetch.go

```diff
@@ -117,6 +119,8 @@ func (f *fetchCmd) run() error {
 		Keyring:  f.keyring,
 		Verify:   downloader.VerifyNever,
 		Getters:  getter.All(settings),
+		Username: f.username,
```

Comment:
It looks like the `--username` and `--password` flags have not been added to this command. If those were left off by mistake, would you mind adding those in?

Reply:
You're right. They were left out by mistake.
I've just added them.